dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -584,7 +597,10 @@ boolean isUnusedClassSymbolReference(ClassSymbolReference reference) {
           constantPoolIndexForClass(sourceJavaClass, targetClassName);
       Verify.verify(
           !targetConstantPoolIndices.isEmpty(),
-          "The target class symbol reference is not found in source class");
+          "The target class symbol reference "
```

Comment:
delete "symbol reference" here. It's understood.

Reply:
Updated